src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHost.cs

```diff
@@ -340,6 +349,7 @@ private void DisposeAndClearSubscriptions()
 
             _evaluationSubscriptionLinks.Clear();
```

Comment:
We also need to think about the fact where user has multi-TFM project, we remove one TF from this list and then add it back. Will everything work fine if we remove the subscription in first step and add it back again later? I'll file a separate issue to track this.